clang/lib/Basic/Warnings.cpp

```diff
@@ -70,6 +76,16 @@ void clang::ProcessWarningOptions(DiagnosticsEngine &Diags,
   else
     Diags.setExtensionHandlingBehavior(diag::Severity::Ignored);
 
+  if (!Opts.DiagnosticSuppressionMappingsFile.empty()) {
+    if (auto Buf =
```

Comment:
This is typically called File and sometimes called FileOrError

Reply:
sure, i think it's a little bit unfortunate to spend so much effort in a variable that's scoped to 2 lines.